example/helm/values-standalone-agent-ngt.yaml

```diff
@@ -33,7 +33,7 @@ agent:
     # Check duration of automatic indexing.
     auto_index_duration_limit: 1m
     # The limit duration of automatic indexing.
-    auto_index_duration_limit: 10s
+    auto_index_check_duration: 10s
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
The comments (L33, L35) should be swapped.